cephfs-provisioner.go

```diff
@@ -425,7 +421,7 @@ func main() {
 
 	// The controller needs to know what the server version is because out-of-tree
 	// provisioners aren't officially supported until 1.5
-	serverVersion, err := clientset.Discovery().ServerVersion()
+	// serverVersion, err := clientset.Discovery().ServerVersion()
```

Comment:
Drop this and above comments while we can remember it's not needed?

Reply:
oh yup, removed